tests/conftest.py

```diff
@@ -1200,6 +1198,80 @@ def cg_dir(tmpdir_factory) -> Path:
     return tmpdir_factory.mktemp("cg")
 
 
+@pytest.fixture(scope="function", name="swegen_dir")
```

Comment:
```suggestion
@pytest.fixture(name="swegen_dir")
```
Default scope is "function" so I think it is unnecessary to supply it